internal/pkg/describe/app.go

```diff
@@ -68,3 +69,26 @@ func (a *App) HumanString() string {
 	writer.Flush()
 	return b.String()
 }
+
+// AppDescriber retrieves information about an application.
```

Comment:
Is the plan to implement the `AppDescriber` and `UpgradeApplication` methods in a separate PR?

Reply:
Yeah